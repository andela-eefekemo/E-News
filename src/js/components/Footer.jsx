import React from 'react';

// Footer component
const Footer = () => (
  <div className="footer">
    <p className="center-align">&copy; 2017 | E-News</p>
    <p className="center-align">Developed by: Efekemo Eguonoghene</p>
  </div>
);

export default Footer;
