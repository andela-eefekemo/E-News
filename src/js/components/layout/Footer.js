import React from 'react';

export default class Footer extends React.Component {
  render() {
    return (
      <div className="footer">
        <p className="center-align">&copy; 2017 | Budget Analysis Application</p>
        <p className="center-align">Developed by: Efekemo Eguonoghene</p>
      </div>
    );
  }
}
