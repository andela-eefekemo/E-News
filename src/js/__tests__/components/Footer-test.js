/* global expect jest test */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../components/Footer';

jest.dontMock('../../components/Footer');

describe('Footer Component', () => {
  test('should match the footer snapshot', () => {
    const component = shallow(<Footer />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
