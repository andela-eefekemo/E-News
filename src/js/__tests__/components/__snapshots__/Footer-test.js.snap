// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Match the Footer component 1`] = `
<div
  className="footer"
>
  <p
    className="center-align"
  >
    Â© 2017 | E-News
  </p>
  <p
    className="center-align"
  >
    Developed by: Efekemo Eguonoghene
  </p>
</div>
`;
